plugins/generic/usageStats/PKPUsageStatsPlugin.inc.php

```diff
@@ -186,6 +185,10 @@ function callbackLoadCategory($hookName, $args) {
 			$this->import('UsageStatsReportPlugin');
 			$plugin = new UsageStatsReportPlugin();
 		}
+		if ($category ==  'blocks') {
+			$this->import('UsageStatsOptoutBlockPlugin');
+			$plugin = new UsageStatsOptoutBlockPlugin($this->getName());
```

Comment:
I didn't see that before, but this block will always appear, no matter if the manager has activated the data privacy option or not right? Is this what you wanted?